diagnostics/top_heaviness_metric/doc/top_heaviness_metric.rst

```diff
@@ -0,0 +1,90 @@
+Top-Heaviness Metric Diagnostic Documentation
+================================
+
+Last update: 5/30/2021
+
+The vertical profiles of diabatic heating have important implications for large-scale dynamics, especially for the coupling between the large-scale atmospheric circulation and precipitation processes. We adopt an objective approach to examine the top-heaviness of vertical motion (Back et al. 2017), which is closely related to the heating profiles and a commonly available model output variable. The diagnostic metric can also be used to evaluate the diabatic heating profile.
+
+Version & Contact info
+----------------------
+
+.. '-' starts items in a bulleted list:
+   https://docutils.sourceforge.io/docs/user/rst/quickref.html#bullet-lists
+
+- Version/revision information: version 1.0 (5/21/2021)
+- Developer/point of contact (Jiacheng Ye, jye18@illinois.edu, DAS UIUC; Zhuo Wang, zhuowang@illinois.edu, DAS UIUC)
+
+.. Underline with '^'s to make a third-level heading.
+
+Open source copyright agreement
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The MDTF framework is distributed under the LGPLv3 license (see LICENSE.txt).
+
+Functionality
+-------------
+
+The currently package consists of following functionalities:
+
+(1) Calculation of the fractional variance of vertical velocity at each grid point explained by two base functions (i.e., idealized deep convection profile, idealized deep stratiform profile)
+
+(2) Calculation of the top-heaviness ratio (O2/O1)
+
+(3) Other analysis (to be added soon)
+
+(**) cropping.py can be referenced if code is needed to either shift the grid of your data
+or to crop your data to a specified region
+
+As a module of the MDTF code package, all scripts of this package can be found under
+``mdtf/MDTF_$ver/diagnostics/top_heaviness_ratio``
+
+Required programming language and libraries
+-------------------------------------------
+
+Python3 packages: "netCDF4", "xarray", "numpy", "scipy", "matplotlib", "basemap"
+
+Required model output variables
+-------------------------------
+
+3-D spatial dimension Omega (units: Pa/s), which can be either the monthly mean in a certain year or the long-term monthly mean (or seasonal) mean.
```

Comment:
Please provide CMOR variable names of the required fields.

Reply:
Thank you for your suggestion. We just made a revision.